PWGLF/Tasks/lambdakzeroanalysisMC.cxx

```diff
@@ -61,19 +61,28 @@ struct lambdakzeroQA {
   HistogramRegistry registry{
     "registry",
     {
-      {"hMassK0Short", "hMassK0Short", {HistType::kTH1F, {{3000, 0.0f, 3.0f}}}},
-      {"hMassLambda", "hMassLambda", {HistType::kTH1F, {{3000, 0.0f, 3.0f}}}},
-      {"hMassAntiLambda", "hMassAntiLambda", {HistType::kTH1F, {{3000, 0.0f, 3.0f}}}},
+      {"hMassK0ShortMCportion", "hMassK0ShortMCportion", {HistType::kTH1F, {{800, 0.0f, 3.0f}}}},
```

Comment:
Here you could consider adding the units, it will for sure pay off in the future